src/driver-hidpp20.c

```diff
@@ -90,7 +90,7 @@ hidpp20drv_read_button_1b04(struct ratbag_button *button)
 
 	control = &drv_data->controls[button->index];
 	mapping = control->control_id;
-	if (control->reporting.divert || control->reporting.persist)
+	if (control->reporting.remapped)
```

Comment:
I have strictly no ideas if this is correct or not.
The original code looked for the HW state while the new seems to be looking at the internal libratbag state. So definitively, I can't provide a good reason to nack/ack this.

Reply:
I don't think that's correct. This field is  [filled in by the hardware](https://github.com/libratbag/libratbag/blob/master/src/hidpp20.c#L1032), I noticed it in the debug output.

Reply:
OK, rereading the doc now, and looks like you are correct. That is some complex remapping to map in ones head.